runtests.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+import os
+import sys
+
+import django
+from django.conf import settings
+from django.test.utils import get_runner
+
+
+if __name__ == "__main__":
+    os.environ['DJANGO_SETTINGS_MODULE'] = 'tests.test_settings'
+    django.setup()
+    TestRunner = get_runner(settings)
+    test_runner = TestRunner()
+    failures = test_runner.run_tests(["tests", "gdpr"])
```

Comment:
Tři předchozí řádky by se daly sloučit do:
`failures = get_testrunner(settings)().run_tests(['tests', 'gdpr'])`